maturity-level-two/src/Controllers/v1/CustomizationController.cs

```diff
@@ -117,13 +115,15 @@ public async Task<IActionResult> Create(NewCustomizationDto newCustomization)
         [SwaggerResponse((int)HttpStatusCode.InternalServerError, "API is not available")]
         public async Task<IActionResult> DeleteCustomization(int id)
         {
-            var CustomizationObj = await _coditoRepository.GetCustomizationAsync(id);
-            if (CustomizationObj == null)
+            int numberOfChanges = await _coditoRepository.DeleteCustomizationAsync(id);
+            if(numberOfChanges > 0)
             {
-                return NotFound(new ProblemDetailsError(StatusCodes.Status404NotFound));
+                return NoContent();
             }
-            await _coditoRepository.DeleteCustomizationAsync(id);
-            return NoContent();
+            else
+            {
+                return NotFound();
+            }           
```

Comment:
IMHO, we should not differentiate here; but that is mentionned in naother issue

Reply:
does this code return the response formatted as problem+json?

Reply:
My bad, i'm replacing this in the Delete and Sell methods.

return NotFound(new ProblemDetailsError(StatusCodes.Status404NotFound))

Reply:
We are normally returning problem+json for all the 4XX and 5XX.
```Csharp
return Conflict(); 
```
should be problem+json